src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSiteJenkinsTest.java

```diff
@@ -24,388 +24,122 @@
 package jp.ikedam.jenkins.plugins.updatesitesmanager;
 
 
+import com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException;
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
 import hudson.model.UpdateSite;
-import jenkins.model.Jenkins;
-
-import org.jvnet.hudson.test.HudsonTestCase;
+import org.hamcrest.Matcher;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.TestExtension;
 import org.jvnet.hudson.test.recipes.LocalData;
 import org.kohsuke.stapler.DataBoundConstructor;
 
-import com.gargoylesoftware.htmlunit.FailingHttpStatusCodeException;
-import com.gargoylesoftware.htmlunit.html.HtmlForm;
-import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.endsWith;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.isA;
 
 /**
  * Tests for DescribedUpdateSite, concerned with Jenkins.
  */
-public class DescribedUpdateSiteJenkinsTest extends HudsonTestCase
-{
-    public static class DescribedUpdateSiteForConfigureTest extends DescribedUpdateSite
-    {
+public class DescribedUpdateSiteJenkinsTest {
+
+    @Rule
+    public JenkinsRule jRule = new JenkinsRule();

```

Comment:
also as we manage static state of whole jenkins, it much more safer to use rule instead of class rule